tests/test_classifiers.py

```diff
@@ -219,3 +219,45 @@ def test_image_classifier_vit():
 
     losses = img_classifier.forward_train(imgs, label)
     assert losses['loss'].item() > 0
+
+
+def test_image_classifier_tnt():
```

Comment:
Remove this unit test. Place all backbone unit tests in `tests/test_models/test_backbones`